<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="question_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Question:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>Number of Answers</string>
   </property>
   <widget class="QRadioButton" name="two_answers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Two Answers</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="three_answers">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Three Answers</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="four_answers">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Four Answers</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>61</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Correct</string>
   </property>
   <widget class="QRadioButton" name="correct_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="correct_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="correct_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="correct_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Answers:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="topic_spin">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>351</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Topic ID:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>two_answers</sender>
   <signal>toggled(bool)</signal>
   <receiver>answer3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>two_answers</sender>
   <signal>toggled(bool)</signal>
   <receiver>answer4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>three_answers</sender>
   <signal>toggled(bool)</signal>
   <receiver>answer3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>three_answers</sender>
   <signal>toggled(bool)</signal>
   <receiver>answer4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>four_answers</sender>
   <signal>toggled(bool)</signal>
   <receiver>answer3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>four_answers</sender>
   <signal>toggled(bool)</signal>
   <receiver>answer4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>two_answers</sender>
   <signal>toggled(bool)</signal>
   <receiver>correct_3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>two_answers</sender>
   <signal>toggled(bool)</signal>
   <receiver>correct_4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>three_answers</sender>
   <signal>toggled(bool)</signal>
   <receiver>correct_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>three_answers</sender>
   <signal>toggled(bool)</signal>
   <receiver>correct_4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>four_answers</sender>
   <signal>toggled(bool)</signal>
   <receiver>correct_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>four_answers</sender>
   <signal>toggled(bool)</signal>
   <receiver>correct_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
